app/controllers/api/v1/patients_controller.rb

```diff
@@ -27,8 +42,24 @@ def patients_params
         :created_at,
         :updated_at,
         phone_numbers: [permitted_phone_number_params],
-        address: permitted_address_params
+        address:       permitted_address_params
       )
     end
   end
-end
+
+  def latest_record_timestamp
+    if params[:first_time].present? && params[:first_time] == 'true'
```

Comment:
I'd rather call this `sync_all`, as it's more descriptive of what's actually happening.